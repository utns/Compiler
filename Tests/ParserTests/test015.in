var
    i, a: integer;

begin
    for i := 0 to 10 do
        a := 1;    
end.
