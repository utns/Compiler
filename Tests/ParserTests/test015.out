begin
for i
____0
to
___10
     ____1
___:=
     ____a
end
